components/timezone/src/time_zone.rs

```diff
@@ -81,6 +82,28 @@ impl CustomTimeZone {
         }
     }
 
+    /// Creates a new [`CustomTimeZone`] representing Greenwich Mean Time
+    /// (London Time as observed in the winter).
+    pub const fn gmt() -> Self {
```

Comment:
Issue: why are there special constructors for two time zones? Our decision to not have constants for particular locales should extend to this.

Reply:
It takes about 10 lines of code to build these manually which takes focus away from the things I'm trying to show in docs strings. When we get a proper time zone class I can use that, but we don't have a proper time zone class. I don't want to bias certain locales so I picked GMT/BST since they are the time zone most closely associated with UTC

Reply:
BST is not that close to UTC, it's a locale time zone used by three countries that has a UTC offset of +1. If you need example time zones for docs you can fake them in `# ` code.

Also if this is just for docs code, they shouldn't have FFI. I don't want to end up in a situation like with `create_en` and `create_bn`.

Reply:
ok, for examples I guess I can stick with UTC

I am not a fan of too much # code because it makes the examples not reproducible.